src/VisualStudio/NuGet.Packaging.VisualStudio/Views/PackageMetadataViewModel.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Packaging.VisualStudio
+{
+	class PackageMetadataViewModel
```

Comment:
Some properties are incorrect. Please ensure they match the ones used in the [build targets](https://github.com/NuGet/NuGet.Build.Packaging/blob/dev/src/Build/NuGet.Build.Packaging.Tasks/NuGet.Build.Packaging.targets#L92-L113)